src/coreclr/md/enc/sha256.cpp

```diff
@@ -0,0 +1,183 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+//*****************************************************************************
+// sha256.cpp
```

Comment:
Having our own implementation of sha256 is going to be compliance burden. You should discuss this with the security folks (email `.NET Security Team`).

Reply:
That's fair. Will do.

Before, I had the impl using the built-in bcrypt for windows, but I do not know an equivalent for the other OSes, other than OpenSSL. I think using OpenSSL might be overkill for this, but is there any other alternative?